src/sentry/static/sentry/app/styles/space.tsx

```diff
@@ -1,28 +1,18 @@
-const space = (size: number): string => {
-  switch (size) {
-    //Our spacing scale is based on a base unit of 8
-    //We use a case switch to prevent odd numbers, decimals, and big numbers.
-    case 0:
-      return '0';
-    case 0.25:
-      return '2px';
-    case 0.5:
-      return '4px';
-    case 0.75:
-      return '6px';
-    case 1:
-      return '8px';
-    case 1.5:
-      return '12px';
-    case 2:
-      return '16px';
-    case 3:
-      return '20px';
-    case 4:
-      return '30px';
-    default:
-      return '8px';
-  }
-};
+const SPACES = {
+  0: '0',
+  0.25: '2px',
+  0.5: '4px',
+  0.75: '6px',
+  1: '8px',
+  1.5: '12px',
+  2: '16px',
+  3: '20px',
+  4: '30px',
+} as const;
+
+type ValidSize = keyof typeof SPACES;
+
+const space = (size: ValidSize): typeof SPACES[ValidSize] =>
```

Comment:
If the parameter is constrained to ValidSize can the 8px case be reached anymore?

Reply:
heh, good point :)

I think we probably should kill that case since it's somewhat unexpected